feature/login.feature

```diff
@@ -1,52 +1,51 @@
 
 Scenario: Login incorreto
     Given o usuário está na página de "login"
-    And existe um usuário cadastrado
-    When eu tento acessar o aplicatico com o cpf "11111111111"
-    And o campo da senha é preenchido por "123456"
-    Then eu recebo uma mensagem  "Credenciais incorretas"
-    And eu continuo na página de login
+    And existe um usuário de CPF "12989087064" cadastrado no sistema
+    When o usuário tenta fazer login com o email "usererror@gmail.com" e senha "1234567a"
+    Then o usuário recebe uma mensagem  "Credenciais incorretas."
+    And o usuário continua na página de "login"
 
+Scenario: Esqueceu a senha
+    Given o usuário está na página de "login"
+    When o usuário seleciona a opção de esqueceu a senha
+    Then o usuário é encaminhado para a pagina de "esqueci a senha" 
 
-Scenario: Esqueceu a senha e preenche o campo de recuperar com o código certo
-    Given o usuário "acruchi" está na página de "login"
-    When o usuário "acruchi" seleciona a opção de Esqueceu a senha
-    Then o usuário "acruchi" é encaminhado para uma pagina de esqueci a senha
-    And preenche o campo de cpf
-    And seleciona recuperar a senha 
-    Then o usuário "acruchi" é encaminhado para a pagina de recuperar a senha
-    And o usuário "acruchi" recebe o código
-    When o usuário "acruchi" preenche o campo do código com o código correto
-    Then o usuário "acruchi" recebe uma mensagem "digite uma nova senha"
-    And o usuário "acruchi" digita uma nova senha
-    Then é encaminhado para a pagina de login 
+Scenario: Recuperar a senha correta
+    Given o usuário está na página de "esqueci a senha"
+    When o usuário preenche o "email" corretamente
+    And recebe uma mensagem "Código enviado para o email"
+    Then é encaminhado para a página de "recuperar a senha"
+  
+Scenario: Recuperar a senha incorreta
+    Given o usuário está na página de "esqueci a senha"
+    When o usuário preenche o email "usererror@gmail.com" 
+    Then recebe uma mensagem "Email inválido"
 
-Scenario: Esqueceu a senha e preenche o campo de recuperar com o código errado
-    Given o usuário "acruchi" não está logado  na aplicação
-    And está na página de "login"
-    When o usuário "acruchi" seleciona a opção de Esqueceu a senha
-    Then o usuário "acruchi" é encaminhado para uma pagina de esqueci a senha
-    And preenche o campo de cpf
-    And seleciona recuperar a senha 
-    Then o usuário "acruchi" é encaminhado para a pagina de recuperar a senha
-    And o usuário "acruchi" recebe o código
-    When o usuário "acruchi" preenche o campo do código com o código "3333"
-    Then o usuário "acruchi" recebe uma mensagem "Código errado"
-    And continua na pagina de recuperar a senha 
+Scenario: Mudando a senha correta
+    Given o usuário está na pagina de "recuperar a senha"
+    When o usuário preenche o "código" corretamente
+    Then o usuário recebe uma mensagem "digite uma nova senha"
```

Comment:
You need to specify the user action for filling in the password field. In this case, you will also have an environment variable for this new password. Remember to add the user action of confirm too.

Reply:
This comment also applies to the scenario below.